updates/20180619-00_update_utf8.sql

```diff
@@ -0,0 +1,74 @@
+ALTER TABLE `item_randomsuffix_groups` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `locales_gossip_menu_option` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `locales_worldmap_info` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `locales_worldstring_table` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_creatures` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_fishing` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_gameobjects` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_items` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_pickpocketing` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `loot_skinning` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `npc_gossip_textid` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `npc_text` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `petdefaultspells` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `playercreateinfo` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `playercreateinfo_bars` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `playercreateinfo_items` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `playercreateinfo_skills` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `playercreateinfo_spells` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `points_of_interest` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `quest_properties` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `reputation_creature_onkill` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `reputation_faction_onkill` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `reputation_instance_onkill` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spell_disable` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spell_disable_trainers` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spell_effects_override` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spell_teleport_coords` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spellclickspells` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spelloverride` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `spelltargetconstraints` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `vehicle_accessories` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `vendor_restrictions` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `vendors` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `wordfilter_character_names` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `wordfilter_chat` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `world_db_version` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `worldmap_info` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `worldstate_templates` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `zoneguards` CHARSET=utf8 COLLATE utf8_unicode_ci;
+ALTER TABLE `world_db_version` CHARSET=utf8 COLLATE utf8_unicode_ci;
+
+UPDATE `character_db_version` SET LastUpdate = '20180619-00_update_utf8';
```

Comment:
It is the world db.... it is "INSERT INTO `world_db_version` VALUES ('32', '20180619-00_update_utf8');"